scripts/main/lychee.js

```diff
@@ -1034,14 +1034,23 @@ lychee.load = function (autoplay = true) {
 			$("#sensitive_warning").removeClass("active");
 			if (album.json && albumID === album.json.id) {
 				if (album.isSearchID(albumID)) {
-					// We are probably coming back to the search results from
-					// viewing an image.  Because search results is not a
-					// regular album, it needs to be treated a little
-					// differently.
-					header.setMode("albums");
-					lychee.setMetaData(lychee.locale["SEARCH_RESULTS"]);
+					if ($(".class_settings").length > 0) {
```

Comment:
I would rather use
```suggestion
					if ($(".settings_view, .users_view, .sharing_view, .logs_diagnostics_view, .u2f_view").length > 0) {
```
And not use `.class_settings` as it is an empty css selector (i.e. it does not provide style).

Reply:
Thank you for taking the time to review. 
I have made the changes.